projects/core/src/user/facade/user.service.ts

```diff
@@ -42,7 +51,13 @@ export class UserService {
    * Loads the user's details
    */
   load(): void {
-    this.store.dispatch(new UserActions.LoadUserDetails(USERID_CURRENT));
+    this.authService
+      .getOccUserId()
+      .pipe(take(1))
+      .subscribe(storefrontUserId =>
+        this.store.dispatch(new UserActions.LoadUserDetails(storefrontUserId))
+      )
+      .unsubscribe();
```

Comment:
To reviewer: pay special attention to this modification because we will be using this in all the user facade functions that need to pass down the userId.